action-binding-generator/src/main/kotlin/io/github/typesafegithub/workflows/actionbindinggenerator/typing/TypingGeneration.kt

```
@@ -76,8 +76,12 @@ private fun EnumTyping.buildEnumCustomType(
     fieldName: String,
     className: String,
 ): TypeSpec {
-    val itemsNames = itemsNames ?: items.map { it.toPascalCase() }
-    val itemsNameMap = items.zip(itemsNames).toMap()
+    val itemsNames =
+        itemsNames ?: items
+            .map { it.toPascalCase() }
+            .groupBy { it }
+            .values
```

One more consideration: since the order of possible enum values doesn't matter, we should be resilient to reordering to not renumber the items. Today, `bam-123` will be accessible under `Bam123_2`, and tomorrow under `Bam123_3`, which may make some workflow either fail or silently succeed in an incorrect way.

How about sorting the values?

---

Order in the YAML should be preserved I think, but yeah, why not.
Sorting is added.